src/__tests__/type-syncer.test.ts

```diff
@@ -139,12 +146,35 @@ function buildSyncer() {
         case 'packages/package-2/package.json':
           return package2File
         default:
-          throw new Error('What?!')
+          throw new Error(`Who?! ${filepath}`)
       }
     }),
     writePackageFile: jest.fn(() => Promise.resolve()),
   }
 
+  const workspaceResolverService: IWorkspaceResolverService = {
+    getWorkspaces: jest.fn(async (root, globber) =>
+      (
+        await Promise.all(
+          (
+            (await (async (root) => {
```

Comment:
Let's refactor this, the `(await (async` is a bit jarring. It's ok to use a regular loop here since there's no real async stuff going on that needs to be concurrent.

Reply:
Yeah. I wasn't pleased about it, but I didn't want to extract it out.
Will do!

Reply:
Is it better now?

Reply:
Much better!